core/src/test/scala/akka/persistence/cassandra/CassandraLifecycle.scala

```diff
@@ -73,15 +100,19 @@ trait CassandraLifecycle extends BeforeAndAfterAll { this: TestKitBase with Suit
     super.beforeAll()
   }
 
-  def startCassandra(): Unit = {
-    val cassandraDirectory = new File("target/" + systemName)
-    CassandraLauncher.start(
-      cassandraDirectory,
-      configResource = cassandraConfigResource,
-      clean = true,
-      port = 0,
-      CassandraLauncher.classpathForResources("logback-test.xml")
-    )
+  def startCassandra(port: Int = 0): Unit = {
```

Comment:
this might not be source compatible for Java? add separate overloaded method instead

Reply:
Done